src/backend/core/api/permissions.py

```diff
@@ -50,5 +53,21 @@ class AccessPermission(IsAuthenticated):
 
     def has_object_permission(self, request, view, obj):
         """Check permission for a given object."""
+        if isinstance(obj, models.Invitation):
+            team = get_object_or_404(models.Team, id=view.kwargs["team_id"])
+            return team.get_abilities(request.user).get("manage_accesses")
+
         abilities = obj.get_abilities(request.user)
         return abilities.get(request.method.lower(), False)
```

Comment:
It feels weird introducing a custom logic using an if in a generic classes used by all the models. Could we extend the  default classes, to respect the [open-close principle](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle)

Reply:
it seems duplicated with the `RelatedTeamAccessPermission` classes that inherit from `IsAuthenticated`

Reply:
Would it be too smart if we test:
```python
if view.kwargs.get("team_id")
```
If we have this kwarg, it means that we are in an url nested under a team? Seems ok and generic to me with a comment to explain.

Reply:
I'd like to keep a dedicated permission for Invitation, mostly for the custom error message (and the linter doesn't appreciate the error message being anywhere else than at class level). 

I'll modify what I need in InvitationPermission then, as it seem that TeamAccesses don't have the same problem 🤔